src/rnp/rnp.c

```diff
@@ -666,7 +667,7 @@ main(int argc, char **argv)
 
     rnp_params_free(&rnp_params);
 
-    if (!rnp_key_store_load_keys(&rnp, rnp_cfg_getint(&cfg, CFG_NEEDSSECKEY))) {
+    if (!rnp_key_store_load_keys(&rnp, (bool) rnp_cfg_getint(&cfg, CFG_NEEDSSECKEY))) {
```

Comment:
Yeah, was thinking about it. Now it seems to be much more applicable for a number of cfg variables.